src/main/java/com/booksaw/betterTeams/util/FoliaUtils.java

```diff
@@ -0,0 +1,56 @@
+package com.booksaw.betterTeams.util;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+
+import static com.booksaw.betterTeams.Main.plugin;
+
+public class FoliaUtils {
+
+	public static boolean isFolia() {
```

Comment:
best to do something like:
static IS_FOLIA = getFolia()
private getFolia() --> this code
public isFolia() return IS_FOLIA

Especially if you need to do this often, it saves a lot of cycles.